basicsettings.ml

```diff
@@ -37,21 +37,7 @@ let welcome_note = Settings.add_string ("welcome_note",
  | |   | | , \\| |   /  \\  \\\n\
  | |___| | |\\ \\ | |\\ \\ _\\  \\\n\
  |_____|_|_| \\__|_| \\_|____/\n\
-Welcome to Links with session types", `System)
-
-(* (* Alternative banner designs *)
-"  _     _ __    _ _  __  ___
- / |   | |  \\  |   |/ / / ._\\
- | |   | | , \\ | |   /  \\  \\
- | |___| | |\\ \\  | |\\ \\__\\  \\
- |_____._._| \\___|_| \\______/",
-"                                         _
-  _     _ __    _ _  _ ___        <>    |_>
- / |   | |  \\  | | |'// ._\\      //     |  
- | |   | | , \\ | |  / \\  \\      //      |
- | |___| | |\\ \\  | | \\_\\  \\  ,-//      _|_
- |_____._._| \\___|_|\\_____/  \\_/      <_|_>"
-*)
+Welcome to Links with session types and shredding", `System)
```

Comment:
I just had a thought that perhaps instead of modifying Links' welcome message we should have a `--version` switch that displays information about features of the current build.